styles/main.css

```diff
@@ -1,74 +1,76 @@
-.reset
-{
-	padding: 0px;
-	margin: 0px;
-	text-align: center;
+body, html {
+	margin: 0;
+	padding: 0;
+	width: 100%;
+	height: 100%;
+	display: table;
 }
 
-.main-but
-{
+body {
+	display: table-cell;
+	vertical-align: middle;
 	text-align: center;
-	margin: auto;
-	border-radius: 50%;
-	height:100px;
-	width:100px;
-	border:5px solid black;
-	line-height: 100px;
-	margin-bottom: 10px;
-	margin-top: 10px;
-}
-.main-but:hover
-{
-	background-color: #ba68c8;
-}
-.main-but--disabled
-{
-	background-color: gray;
-	color:white;
-}
-.main-but--normal
-{
-	background-color: #f06292;
-	color:white;
+	background-color: #1e1e1e;
 }
-.main-but--success
-{
-	background-color: green;
-	color:white;
-}
-
 
-.login
-{
-	margin-top: 10%;
+.sr-only {
+	display: none;
 }
 
+.login input {
```

Comment:
for CSS i recommend BEM https://abdurrahmaanjanhangeer.wordpress.com/2018/02/18/bem-simple-css-naming-approach/ and BEM discourages general reset. fine for now